lib/dradis/plugins/pdf_export/processor.rb

```diff
@@ -0,0 +1,161 @@
+module Dradis
+  module Plugins
+    module PdfExport
+      class Processor < Prawn::Document
+        def initialize(args={})
+          super(top_margin: 70)
+
+          content_service = args[:content_service]
+
+          @author = 'Security Tester'
+          @email  = 'tester@securitytesting.com'
+          @issues = content_service.all_issues
+          @notes  = content_service.all_notes
+          @title  = "Dradis Framework - v#{Dradis::CE::VERSION::STRING}"
+
+          sort_issues
+        end
+
+        def generate
+          cover_page
+          project_notes
+          summary_of_findings
+          detailed_findings
+          tool_list
+
+          # outline
```

Comment:
are we not using the `outline` method? should it be removed?

Reply:
ok...